### ISSUE TITLE
Synthesizer loss increases/diverges under training with GPU

### ISSUE THREAD

**Issue by tcchau** [babysor/MockingBird#612]
**Summary[问题简述（一句话）]**
If I use CPU to train the synthesizer, under the fine-tuning methodology, I get good results and the loss has been decreasing over time. However, when I moved the models over to an Ubuntu container, running ROCm for GPU acceleration using the AMD graphics cards, the loss actually diverges.

Has anyone else experienced this, and if so, how did you solve it?

**Env & To Reproduce[复现与环境]**
描述你用的环境、代码版本、模型
Ubuntu 20.04
ROCm 5.1, using RX580
Pytorch 1.11
aidatatang_200zh

**Screenshots[截图（如有）]**
If applicable, add screenshots to help

**Comment by babysor** [comment]
WTF.... It's quite a funny scene and you should file a bug to AMD.
